src/main/kotlin/no/nav/klage/oppgave/domain/joark/ArkivsaksSystem.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.klage.oppgave.domain.joark
+
```

Comment:
Dersom enumene skal eksponeres i Front-end, så bør de implementere `Kode` interfacet, se på for eksempel `Type`

Reply:
Jeg lurte også litt på det, men tror ikke dette skal eksponeres ut, men kanskje kun brukes til lagring i Joark?

Reply:
I dette tilfellet er det kun brukt til lagring i Joark, som Andreas nevner. `Fagsystem` i `OversendtKlage.kt` ser ut til å stemme greit overens med dette, jeg kan lage en mapping fra den til verdiene jeg sender inn til Joark.